src/common/style/vanilla/footer.scss

```diff
@@ -71,11 +71,6 @@ body,
     background-position: 0 0;
   }
 
-  .p-social-icon--google {
-    @extend %p-social-icon;
-    background-position: -40px 0;
-  }
-
```

Comment:
Don't you need to move the `background-position` for Facebook and Youtube to account for this?  They seem to be positioned at 40-pixel intervals.

Reply:
The sprite image containing the icons doesn't change, so the position of other icons is not changed.